rotkehlchen/globaldb/handler.py

```diff
@@ -1853,23 +1853,26 @@ def get_collection_main_asset(identifier: str) -> Optional[str]:
         return result[0] if result is not None else None
 
     @staticmethod
-    def get_or_write_abi(serialized_abi: str, abi_name: Optional[str] = None) -> int:
+    def get_or_write_abi(
+            write_cursor: 'DBCursor',
```

Comment:
Hmmm good question. There definitely was a problem with cursors, but by checking the code now I don't see it. Probably I've changed some things since that time. Will revert this change :+1:

Reply:
But you have not yet reverted this change. So what's the plan? not do it here?

Reply:
Ouchhhhh, I wrote that I will do it, but didn't do :sob: